src/content/advanced/private-clusters/index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Private clusters
+description: By default, the Kubernetes API endpoint is public and the cluster has internet access. This article shows which products offer private cluster features.
```

Comment:
```suggestion
description: Learn how to manage private clusters in Giant Swarm. A private cluster lets you limit the Kubernetes API access and at the same time control egress traffic of your workload using a proxy.
```
I would not assume a default since we might change in future to `private` as default cluster type